temporal-sdk/src/test/java/io/temporal/worker/StickyWorkerTest.java

```diff
@@ -489,13 +490,18 @@ public void workflowsCanBeQueriedAfterEviction() throws Exception {
     private TestWorkflowEnvironment testEnv;
     private WorkerFactory factory;
     private final String identity = UUID.randomUUID().toString();
+    private final String binaryChecksum = UUID.randomUUID().toString();
 
     public TestEnvironmentWrapper(WorkerFactoryOptions options) {
       if (options == null) {
         options = WorkerFactoryOptions.newBuilder().build();
       }
       WorkflowClientOptions clientOptions =
-          WorkflowClientOptions.newBuilder().setNamespace(NAMESPACE).setIdentity(identity).build();
+          WorkflowClientOptions.newBuilder()
+              .setNamespace(NAMESPACE)
+              .setIdentity(identity)
+              .setBinaryChecksum(binaryChecksum)
```

Comment:
Does test verify that it was indeed added to the history?

Reply:
I've added a separate test for it in the WorkflowTest, this one just adds it because identity was there already.